tokenserver-auth/src/oauth/py.rs

```diff
@@ -21,7 +21,7 @@ use std::{sync::Arc, time::Duration};
 pub struct Verifier {
     // Note that we do not need to use an Arc here, since Py is already a reference-counted
```

Comment:
```suggestion
```

and note to myself: [Pyo3's migration guide](https://pyo3.rs/main/migration#pyclone-is-now-gated-behind-the-py-clone-feature) recommended moving to Arc<Py<>>

Reply:
Thanks! forgot to clean up that comment.